Makefile

```diff
@@ -57,8 +57,17 @@ crossdev:
 	cd documents$(S)release$(S)crossdev ; $(CDEL) crossdev.zip ; zip -r -q crossdev.zip *
 
 samples:
-	zip -r -j -q release$(S)samples.zip basic$(S)code basic$(S)images$(S)test$(S)test.gfx basic$(S)images$(S)graphics.gfx
-	zip -d -q release$(S)samples.zip *.tass 
+	# copy BASIC examples into examples/ tree
```

Comment:
It would be easier to just move the basic examples in the repository. I think its much easier to understand when all examples reside there.

Reply:
i agree - to give some directory structure to the release zipfile, after the build, those files will be present under examples/BASIC anyways, as duplicates - probably even those compiling from source would be more likely to notice them under examples/ first